kover-offline-runtime/examples/runtime-api/src/test/kotlin/Tests.kt

```diff
@@ -2,14 +2,29 @@ package org.jetbrains.kotlinx.kover
 
 import kotlinx.kover.offline.runtime.api.KoverRuntime
 import java.io.File
+import java.nio.file.Files
 import kotlin.test.Test
+import kotlin.test.assertContentEquals
 import kotlin.test.assertEquals
 
 class Tests {
     @Test
     fun test() {
         MainClass().readState()
 
+        val reportFile = Files.createTempFile("kover-report-", ".ic").toFile()
+
+        // save binary report to file
+        KoverRuntime.saveReport(reportFile.canonicalPath)
+
+        // get binary report as byte array
+        val bytes = KoverRuntime.getReport()
+
+        // check reports are same
+        val bytesFromFile = reportFile.readBytes()
```

Comment:
I'd also add a test that reads these binary reports back and checks if it does make sense. Because currently, both these methods can write a file/array consisting of garbage, and this test will pass.

Reply:
this test is written to ensure that the methods return the same thing.
What exactly is returned - for this you need to write a more complex infrastructure, which can be done later (ic format now has some unstable parts)